File: medium-101/tip-calculator1.py
total = float(input("What is the total bill?\n"))
service = input ("Level of service (Good, fair, bad)?")

if service == "good": 
    tip = total * .20
elif service == "fair": 
    tip = total * .15
elif service == "bad": 
    tip = total * .10
 
ta = total + tip

print("Tip amount: %.2f" % tip)
print("Total amount: %.2f" % ta)

split = (input("Would you like to split bill?\n"))
persons = int(input("How many persons?"))

if split == "yes":
    amount = float(ta / persons)
print ("Amont per person: %.2f" % amount)

File: javaScript-medium/tip-calculator1.js
// Write a function tipAmount that is given the bill amount and the level of service (one of good, fair and poor) and returns the dollar amount for the tip. Based on:

// good -> 20% fair -> 15% bad -> 10%


let good = .2
let fair = .15
let bad = .1

function tipAmount (total , rate){
    console.log ('Tip amount', ':' , total * rate)
}

tipAmount (100, good)
tipAmount (50, fair)
tipAmount (60, bad)


// function tipAmount (total , service){
//     if (service == "good") {
//         tip = total * 0.2;
//     }else if (service == 'fair'){
//         tip = total * 0.15;
//     }else if ( service == 'bad'){
//         tip = total * .10;
//     }
//     console.log(`Tip Amount is ${tip}`)
// }

// tipAmount (100, "good")

